src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3679,6 +3679,39 @@ public override IOperation VisitInstanceReference(IInstanceReferenceOperation op
             }
         }
 
+        public override IOperation VisitDynamicInvocation(IDynamicInvocationOperation operation, int? captureIdForResult)
+        {
+            if (operation.Operation != null)
+            {
+                _evalStack.Push(Visit(operation.Operation));
+            }
+
+            PushArray(operation.Arguments);
+            ImmutableArray<IOperation> rewrittenArguments = PopArray(operation.Arguments);
+            IOperation rewrittenOperation = operation.Operation == null ? null : _evalStack.Pop();
+
+            return new DynamicInvocationExpression(rewrittenOperation, rewrittenArguments, ((HasDynamicArgumentsExpression)operation).ArgumentNames, ((HasDynamicArgumentsExpression)operation).ArgumentRefKinds, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
+        }
+
+        public override IOperation VisitDynamicIndexerAccess(IDynamicIndexerAccessOperation operation, int? captureIdForResult)
```

Comment:
>VisitDynamicIndexerAccess [](start = 35, length = 25)

It looks like beginning of this function is the same as in VisitDynamicInvocation, consider extracting into a helper method and reusing.